test/prism_regression/literal_integer.parse-tree.exp

```diff
@@ -1,3 +1,13 @@
-Integer {
-  val = "42"
+Begin {
+  stmts = [
+    Integer {
+      val = "42"
+    }
+    Integer {
+      val = "+42"
+    }
```

Comment:
Interesting, I'm surprised this isn't a unary `+` and `-` method call.

Reply:
For `~` it is though. So we'll need to handle it differently when we address those cases.